src/lib/run_environment.ml

```diff
@@ -94,10 +94,37 @@ module Tool = struct
   end
 end
 
+module Make_fun = struct
+  module Requirement = struct
+    type t = [
+      | `Processors of int
+      | `Internet_access
+      | `Memory of [ `GB of float | `Decent | `Big ]
+      | `Quick_run
```

Comment:
I think \`Quick_run means "shouldn't be scheduled"; why not have the converse as a requirement, e.g. \`Scheduler?